schemas/collaborations.yml

```diff
@@ -25,7 +25,8 @@ id: "enum(
 	  'mcdonald-lab',
 	  'mchugh-lab',
 	  'rothman-lab',
-	  'spade-lab'
+	  'spade-lab',
+    'wilhelmus-lab'
```

Comment:
spacing off?

Reply:
Should be fixed now, I merged "main" where it was repaired.

Reply:
confirmed